### ISSUE TITLE
Need the set of islands/regions to aggregate results

### ISSUE THREAD

**Issue by qjhart** [ssj-delta-cu/ssj-overview#10]
One of the requirements for our Consumptive Use providers is that they summarize their ET estimates by crop type and by island/region.   So, we need to provide the set of island/regions that we want to summarize over.   The major islands was what was brought up, but it can include any other method you'd like to see it divided, if there are any political boundaries, you want to include.  Also, it's helpful to have an agreed upon map of what is water (or not water), though I imagine some methods will do their own mask for that as well.

Basically, all I'm looking for is a shapefile/etc that has a unique name and a polygon(s) associated with it.  And then a column I suppose saying if it's water or land.  

| id | Name | water
| 1 | foo island | N
| 2 | bar slough | Y 

I think that's all I need.  The only other aspect is that we can publish that to the public.

**Comment by mclay120** [comment]
I think the shapefile we have is pretty good for this purpose.  Maybe some adjustments to boundaries depending on suggestions in the group.  I will double check with Susan that it is okay to use and share the data.

**Comment by qjhart** [comment]
@ssj-delta-cu/ssj I have moved the part of issue #9 regarding the islands and regions to here.  Also I have assigned this to Mui, who is creating the data layer, that will be shared in this project.  We are in contact with John Collins at waterboards.  In the end I think we will have; 60 core islands and tracts, 2 primary and secondary ares; The point locations of KT's field work and hopefully the water districts.

**Comment by qjhart** [comment]
John has forwarded along this overview, with a better description of the current polys in the primary region.  Can you review these and verify they are consistent with the boundaries you added?

Also, can we make another attribute that is zone, which is either primary or secondary? based on the same overview?

[SacSanJoaquinatltasdelta_overview.pdf](https://github.com/ssj-delta-cu/ssj-overview/files/209013/SacSanJoaquinatltasdelta_overview.pdf)

**Comment by mclay120** [comment]
The majority of boundaries seem consistent with what we have now.  There are a few additional names not in delta watermaster and slight differences in some island boundaries.  If the overview map is the reference, then some renaming and realigning boundaries will need to be done.  The  primary and secondary zone attribute addition is not doable without splitting polygons.  I would like to get the shapefile for the primary and secondary zones.  Actually, would it possible to get all the shapefile data used to produce the maps in the pdf file?
